R/mapping_using_bigQuery_drugExposure.R

```diff
@@ -0,0 +1,118 @@
+# Load necessary libraries
+library(dplyr) # For data manipulation
+library(DBI) # For establishing connection with Database
+library(readr) #To read and write CSV files
+library(AMR)  # For drug name mapping
+
+# Load data from CSV files
+drug_exposure <- read_csv("../data/OMOP/cleaned_drug_exposure.csv")
+concept <- read_csv("../data/OMOP/cleaned_concept.csv")
+
+
+# Print the column names to check if they are correct
+print(colnames(concept))
+# Print the cleaned data to verify everything looks correct
+print(head(concept))
+
+# Print the column names to check if they are correct
+print(colnames(drug_exposure))
+# Print the cleaned data to verify everything looks correct
+print(head(drug_exposure))  
+
+
+#drug_exposure <- drug_exposure %>%
+#  left_join(concept, by = c("drug_concept_id" = "concept_id")) %>%
+#  rename(drug_name = concept_name) %>%
+#  select(-domain_id, -vocabulary_id, -concept_class_id, -standard_concept, -concept_code, 
+#         -valid_start_date, -valid_end_date, -invalid_reason)

```

Comment:
```suggestion
#This block has been commented out as 'route' column is NA and needed to add a line of code to assign route as NA. In the future, this block can be re-used instead of the uncommented one when we have another omop data extraction that has valid 'route' values. 
#drug_exposure <- drug_exposure %>%
#  left_join(concept, by = c("drug_concept_id" = "concept_id")) %>%
#  rename(drug_name = concept_name) %>%
#  select(-domain_id, -vocabulary_id, -concept_class_id, -standard_concept, -concept_code, 
#         -valid_start_date, -valid_end_date, -invalid_reason)

```